spec/hatchet/buildpack_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require_relative '../spec_helper'
+
+RSpec.describe 'Buildpack execution' do
```

Comment:
Is it worth adding an additional single test for another Heroku stack other than the default?

For Python I use a GitHub Actions matrix, but that's probably not worth it for .NET seeing as very little is stack-specific. However, it's probably worth having at least one test in this file to ensure everything works on older stacks.

For examples of setting a specific stack for a test, see:
https://github.com/search?q=repo%3Aheroku%2Fheroku-buildpack-python+%2Fstacks%3A%2F&type=code

Reply:
I was planning to test both `heroku-22` and `heroku-24` stack using a GitHub matrix like the python buildpack, but waited until now to reduce the number of tests executed during development (since, as you point out, there's very little stack-specific that could cause failures).

I probably prefer using the GitHub matrix approach over adding a single test to verify behavior on `heroku-22`. It seems more prudent to execute all tests (limited in numbers as they may be currently) against all stacks we support, even if it's probably safe to assume they'll behave the same.

Also, the stack-specific Hatchet tests make more sense for python, as it verifies something specific to the related stacks (e.g. upgrading/downgrading stacks, and differences in behavior between `heroku-20` and later versions). The integration tests in this repo doesn't verify anything stack-specific like that, so seems to make more sense to handle stack configuration at the runner level :)

I've configure the GitHub matrix for integration tests here: https://github.com/heroku/heroku-buildpack-dotnet/pull/14/commits/ee976731c0288633088f3b37747b9d7cdf5af4f4